src/xmlutils.h

```diff
@@ -36,6 +37,30 @@ namespace libcellml {
  */
 XmlNamespaceMap determineMissingNamespaces(const XmlNamespaceMap &namespaceMap1, const XmlNamespaceMap &namespaceMap2);
 
+/**
+ * @brief Traverse the tree and return an @c XmlNamespaceMap of element namespaces.
+ *
+ * From the root node of the tree given by the @p node, traverse the tree
```

Comment:
Replace with `Traverse the tree from the root @p node`?

Reply:
[f1e2c3a](https://github.com/cellml/libcellml/pull/1225/commits/f1e2c3a294e78530e35c2071d73f9dd1943577ff)